cypress/utils/utils.ts

```diff
@@ -781,7 +781,42 @@ export function deleteAllMigrationWaves(currentPage = false): void {
         });
 }
 
-export function deleteApplicationTableRows(currentPage = false): void {
+export function deleteApplicationTableRows(): void {
```

Comment:
This test is using the deleteApplicationTableRows passing the `currentPage` argument, if you remove it, that is gonna make that test fail.

`ests/migration/applicationinventory/assessment/pagination.test.ts`

Reply:
Removed that test , it is not a valid test case . It is tested at some other places too .